### PR TITLE
[169597711] Gemify Styleguide

### PR DESCRIPTION
**Pull request by sasha-h-reenhanced**
# SUMMARY
Allow using `styleguide` as a gem.

## Possibilities
- [x] install the gem
- [x] use gem dependencies
- [x] include assets
- [x] precompile assets with the project assets
- [x] allow to check `index.html` with all components

### CODE REVIEW

**Comment by sasha-h-reenhanced** [comment]
@lubc @cleicar @nykka please check the PR when you have some time! 🔢

File: README.md [hunk 1]
```diff
@@ -1 +1,50 @@
...
+Or install it yourself as:
+
+    $ gem install ombulabs-styleguide
+
+## Usage
+
+In application.css, add:
+
+```ruby
+*= require ommbulabs/styleguide
```
Comment (lubc) [inline README.md]: @sasha-h-reenhanced typo here

File: README.md [hunk 2]
```diff
@@ -1 +1,50 @@
...
+```
+
+or if you are using scss: appliation.scss
+
+```ruby
+@import "ommbulabs/styleguide";
```
Comment (lubc) [inline README.md]: @sasha-h-reenhanced and here

File: README.md [hunk 3]
```diff
@@ -1 +1,50 @@
...
+```
+
+In application.js add:
+```ruby
+//= require ommbulabs/styleguide
```
Comment (lubc) [inline README.md]: @sasha-h-reenhanced and here  👀

**Review by lubc — APPROVED** [review]
@sasha-h-reenhanced Looks great! 👍 

Only one small typo in the readme file

**Comment by sasha-h-reenhanced** [comment]
@lubc thank you for your review!

**Pull request merged**